### ISSUE TITLE
Customize LED color

### ISSUE THREAD

**Issue by realmain** [moezbhatti/qksms#1035]
Per #651, it looks like LED color does not exist anymore. Is there a reason why it was removed? Can we please re-implement that?

**Comment by moezbhatti** [comment]
Would you strongly prefer to have this over the app automatically assigning the LED color based on the theme?

**Comment by realmain** [comment]
Personal opinion is... why not have both as an option? :)

Some people might like a specific theme for QKSMS, but sometimes the notification light is the same color as other apps. It'd be nice to have the option to manually choose the color of the LED notification so that the user can differentiate a notification between QKSMS and another app.

**Comment by amyers3** [comment]
I would agree that I've never assigned a color of LED or even my own contact colors, but I'm a simple guy. I know lots of users of other programs _love_ customization. If you could come up with a standard theme editor that had colors, led light colors, bubble colors, vibration, etc., I think this could be very clean. Then provide the same access to the panel for each conversation.

Same thing applies to small edits, such as renaming Group messages, people love this... have I used it? No, but you reach that niche group and ex-iPhone users that had some of these features in standard iOS text app.

Customization is king, for most.

**Comment by TechyZeldaNerd** [comment]
I enjoyed using this setting, as well as others such as the ability to change the vibration pattern. While I understand the complexity adding a setting can cause, I really enjoyed the customizability

**Comment by Thovi98** [comment]
I agree, I would love to see this feature !

**Comment by Synntix** [comment]
> Would you strongly prefer to have this over the app automatically assigning the LED color based on the theme?

Personally I'd love to have the LED color following the discussion theme. In my opinion this would be a neat feature without overwhelming the settings

**Comment by Parad0xical** [comment]
I would definitely like to be able to assign a QKSMS notification light colour, as stated by "realmain".  I can do this with Silence but QKSMS defaults to white, which is already used by another app (which also has no options to change LED colour!) As a user of multiple messaging/email apps, it's useful to see which one is providing the alert.

For me, assigning LED colour per contact or per conversation theme would be too much to commit to memory....

**Comment by ghost** [comment]
Abandoned app, closing issue
